ncard_app/models.py

```diff
@@ -49,37 +49,41 @@ class DisplayOnWebsite(models.IntegerChoices):
         YES = 1, 'Yes'
         STUDENT = 2, 'Yes - student'
 
+    class Clinician(models.IntegerChoices):
+        NO = 0, 'No'
+        YES = 1, 'Yes'
+
     title = models.CharField(max_length=16, blank=True)
-    given_name = models.CharField(max_length=64)
-    middle_name = models.CharField(max_length=64, blank=True)
-    surname = models.CharField(max_length=64, blank=True)
+    given_name = models.CharField('Given Name',max_length=64)
```

Comment:
Django expects verbose_names to be lower-case, except for letters which need to be upper-case all the time, such as proper nouns or acronyms.

Reply:
then how can we capitalise the display text (from "Given name" to "Given Name")